ICOTokens.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ICOTokens": [
+    {
```

Comment:
Lets leave this array empty... we cant ship to production with this script hash in the app (its not valid) also this data structure needs to include network id info